### PR TITLE
418 obligation data consistency

### PR DESCRIPTION
**Pull request by alexkiro**
Refs #418 

Should be done, but still need to verify it works as expected and add some tests.

### CODE REVIEW

File: ozone/core/api/views.py [hunk 1]
```diff
@@ -712,9 +726,10 @@ def perform_create(self, serializer):
 
 
 class DataOtherViewSet(SerializerDataContextMixIn, viewsets.ModelViewSet):
+    form_types = None
```
Comment (catalinjitea) [inline ozone/core/api/views.py]: Shouldn't form_types be `other` in this case?

Comment (alexkiro) [reply ozone/core/api/views.py]: Nice catch :smile: , should be fixed in d63c8ee3d468

**Comment by alexkiro** [comment]
Should be finished now, together with test to check that it actually works.

**Comment by sorinstelian** [comment]
Reviewing :)

**Pull request merged**